asyncudp/__init__.py

```diff
@@ -58,7 +59,7 @@ async def recvfrom(self):
         return await self._protocol.recvfrom()
 
 
-async def create_socket(local_addr=None, remote_addr=None):
+async def create_socket(local_addr=None, remote_addr=None, *, queue_max_size=0):
```

Comment:
What value does the `*` add? I mean, I read that at least one core Python thinks it's confusing to new Python users =)

Reply:
`*` do not add any values. In python it is just a mark, that rest of arguments are keyword-only arguments.